bookkeeper-server/src/test/java/org/apache/bookkeeper/client/api/BookKeeperApiTest.java

```diff
@@ -275,11 +276,13 @@ public void testCannotDeleteLedgerTwice() throws Exception {
         result(newDeleteLedgerOp().withLedgerId(lId).execute());
     }
 
-    private static void checkEntries(Iterable<LedgerEntry> entries, byte[] data)
+    private static void checkEntries(LedgerEntries entries, byte[] data)
         throws InterruptedException, BKException {
-        for (LedgerEntry le : entries) {
-            assertArrayEquals(data, le.getEntry());
+        Iterator<LedgerEntry> iterator = entries.iterator();
+        while(iterator.hasNext()) {
+            LedgerEntry entry = iterator.next();
+            // since getEntry() will call ByteBuf release, so no need release here.
```

Comment:
this is clear but maybe it is better to call always call "close".
two motivations:
- close lets the reclycler to work, maybe it is not a real issue
- please use simple test cases as this as a code example

please take into account this comment but it is not a real problem

Reply:
Thanks. will change it.